### DISCUSSION TITLE
Welcome to GodaddyWrapper.Net Discussions!

### DISCUSSION THREAD

**Discussion by ahwm** [ahwm/GodaddyWrapper.Net#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by inzi** [comment]
While it's very quiet in here - I have a need to automate some DNS record creation and updating.

I probably give this project a look see. If I can use it to my project, I'll contribute any updates and fixes.

Cheers, thanks for your work on this.
